packages/cosmic-swingset/lib/block-manager.js

```diff
@@ -126,7 +130,7 @@ export default function makeBlockManager({
     savedActions = currentActions;
     computedHeight = action.blockHeight;
 
-    console.log(
+    log(
```

Comment:
maybe log.info()?  Does anylogger give you the ability to throttle? [e.g. I want to see every 20th message from this source.]

Reply:
As above.

For your second question, you're welcome to implement that in `packages/cosmic-swingset/lib/anylogger-agoric.js`. 😉